init.zsh

```diff
@@ -76,8 +76,16 @@ function pmodload {
   local -a locations
   local pmodule
   local pmodule_location
-  local pfunction_glob='^([_.]*|prompt_*_setup|README*|*~)(-.N:t)'
 
+  # For some reason, Termux fails to locate the "prompt" module when NO_CASE_GLOB
+  # is set, but simply setting CASE_GLOB messes up the prompt itself.
+  # This temporarily enables it for the two globs, then immediately resets it.
+  # Adapted from https://unix.stackexchange.com/a/42031
+  local case_glob_prefix=NO
```

Comment:
Should this be `NO_`?

Reply:
Additionally, we may want to do this check after every module load so we don't accidentally clobber it if it's set in a module.